runtime-watcher/tests/e2e/utils/metrics.go

```diff
@@ -0,0 +1,99 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os/exec"
+	"regexp"
+	"strconv"
+
+	"github.com/kyma-project/runtime-watcher/skr/internal/watchermetrics"
+)
+
+func ExposeSKRMetricsServiceEndpoint() error {
```

Comment:
Any specific reasoning why this has been implemented in Go-Code instead of including this step in the GH-Action pipeline as a step, like here https://github.com/kyma-project/lifecycle-manager/blob/main/.github/workflows/test-e2e.yml#L306

I would prefer if we keep our project in the same shape :D

Reply:
@jeremyharisch I was having issues setting it up using the GitHub action and I thought it was related to the kyma creation. But I think I am wrong. I'll try to move it to the Github Action definition again and see if it is working

Reply:
ah now I remember :D because at this point there is no Kyma-system namespace or the metrics service resource yet. That's why I had this method in the test itself as Go code